src/__tests__/tieCorrection.test.js

```diff
@@ -4,13 +4,13 @@ describe('tie correction', () => {
   it('simple case with two ranks', () => {
     let ranks = [1, 2.5, 2.5, 4];
     let result = tieCorrection(ranks);
-    expect(result).toStrictEqual(0.9);
+    expect(result).toBe(0.9);
   });
 
   it('one element', () => {
     let ranks = [2];
     let result = tieCorrection(ranks);
-    expect(result).toStrictEqual(1);
+    expect(result).toBe(1);
   });
 
   it('zero element', () => {
```

Comment:
i think we must have :
` toThrow('tieCorrection: array length should be greater than 0');`

Reply:
sounds correct

Reply:
I'm talking about the last test
"zero element"

Reply:
I mean your comment sounds correct and the toThrow should be changed.